modules/platform_chassis_extension/manifests/init.pp

```diff
@@ -2,5 +2,11 @@
 class platform_chassis_extension(
   $config
 ) {
-	# Empty, just here as a meta-package.
+	# Add an nginx config file that includes .config/nginx-additions*.conf
+	file { "/etc/nginx/sites-available/${fqdn}.d/altis-additions.nginx.conf":
+		ensure  => present,
+		content => template('platform_chassis_extension/nginx.conf'),
```

Comment:
Could maybe just do this as a straight up inline string here? 🤔

Reply:
Forgot that was a thing!